src/main/java/org/myrobotlab/process/InProcessCli.java

```diff
@@ -386,15 +386,15 @@ public synchronized void stop() {
       worker.interrupt();
     }
 
-    if (in != null) {
+    if (in != null && in != System.in) {
       try {
         in.close();
       } catch (Exception e) {
         log.info("sdin error");
       }
     }
 
-    if (out != null) {
+    if (out != null  && out != System.out) {
```

Comment:
System.out.equals(out)  would be better.

Reply:
k